### DISCUSSION TITLE
Receiving unexpected packet type with ssh2 from sdk.

### DISCUSSION THREAD

**Discussion by fbeyl** [zowe/zowe-cli#1510] [Q&A] [answered]
Tried node sdk 'disk space' example but get ssh2 errors. Every now and then there is a succesfull run but then again several 'type errors'.
```
Outbound: Sending KEXECDH_INIT
Outbound: Sending DISCONNECT (3)
node:events:491
      throw er; // Unhandled 'error' event
      ^
Error: Received unexpected packet type 0
```
Difference I see in client.js is added line 710 proto.start();
When I use ssh2 node-module not from sdk but symlinking to outside ssh2 no more errors occur.
ssh2 version used in sdk is 1.4.0 ---> returns error
outside ssh2 version is 1.11.0 ---> returns ok
 Is this known issue? Anyone an idea?

**Comment by t1m0thyj** [comment, marked as answer]
Resolved in an issue: https://github.com/zowe/zowe-cli/issues/1516
